codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/BuilderGenerator.kt

```diff
@@ -104,6 +106,53 @@ class BuilderGenerator(
         }
     }
 
+    // TODO(EventStream): [DX] Update builders to take EventInputStream as Into<EventInputStream>
```

Comment:
_maybe_. Editors won't generate as helpful type hints in this case, it may be better DX to actually keep an explicit type.